CHANGELOG.md

```diff
@@ -2,6 +2,176 @@
+  to retrieve the IAM policy statement JSON.
+* **iam**:  `AwsPrincipal` has been removed, use `ArnPrincipal` instead.
+* **s3:** `s3.StorageClass` is now an enum-like class instead of a regular
+enum. This means that you need to call `.value` in order to obtain it's value.
+* **s3:** `s3.Coordinates` renamed to `s3.Location`
+* **codepipeline:** `Artifact.s3Coordinates` renamed to `Artifact.s3Location`.
+* **codebuild:** * **codebuild**: buildSpec argument is now a `BuildSpec` object.
+* **lambda:** * **lambda:** `lambda.Runtime.NodeJS*` are now `lambda.Runtime.Nodejs*`
+* **core:** multiple changes to the `Stack` API
+* **core:** `stack.name` renamed to `stack.stackName`
+* **core:** `stack.stackName` will return the concrete stack name. Use `Aws.stackName` to indicate { Ref: "AWS::StackName" }.
+* **core:** `stack.account` and `stack.region` will return the concrete account/region only if they are explicitly specified when the stack is defined (under the `env` prop). Otherwise, they will return a token that resolves to the AWS::AccountId and AWS::Region intrinsic references. Use `Context.getDefaultAccount()` and `Context.getDefaultRegion()` to obtain the defaults passed through the toolkit in case those are needed. Use `Token.isUnresolved(v)` to check if you have a concrete or intrinsic.
+* **core:** `stack.logicalId` has been removed. Use `stack.getLogicalId()`
+* **core:** `stack.env` has been removed, use `stack.account`, `stack.region` and `stack.environment` instead
+* **core:** `stack.accountId` renamed to `stack.account` (to allow treating account more abstractly)
+* **core:** `AvailabilityZoneProvider` can now be accessed through `Context.getAvailabilityZones()`
+* **core:** `SSMParameterProvider` can now be accessed through `Context.getSsmParameter()`
+* **core:** `parseArn` is now `Arn.parse`
+* **core:** `arnFromComponents` is now `arn.format`
+* **core:** `node.lock` and `node.unlock` are now private
+* **core:** `stack.requireRegion` and `requireAccountId` have been removed. Use `Token.unresolved(stack.region)` instead
+* **core:** `stack.parentApp` have been removed. Use `App.isApp(stack.node.root)` instead.
+* **core:** `stack.missingContext` is now private
+* **core:** `stack.renameLogical` have been renamed to `stack.renameLogicalId`
+* **core:** `IAddressingScheme`, `HashedAddressingScheme` and `LogicalIDs` are now internal. Override `Stack.allocateLogicalId` to customize how logical IDs are allocated to resources.
+* **cli:** The CLI no longer accepts `--rename`, and the stack
+                 names are now immutable on the stack artifact.
+* **sns:** using a queue, lambda, email, URL as SNS Subscriber now
+requires an integration object from the `@aws-cdk/aws-sns-subscribers`
+package.
+* **ecs-patterns:** Renamed QueueWorkerService for base, ec2 and fargate to QueueProcessingService, QueueProcessingEc2Service, and QueueProcessingFargateService.
+* * iam: `roleName` in `RoleProps` is now of type `PhysicalName`
```

Comment:
Formatting here looks off

Reply:
Fixed. not sure what happened there 🤷🏻‍♂️